Paper.tex

```diff
@@ -752,6 +752,12 @@ \subsection{Execution}
 $\hyperlink{G__accesslistaddress}{G_{\mathrm{accesslistaddress}}}$ and $\hyperlink{G__accessliststorage}{G_{\mathrm{accessliststorage}}}$ are the costs of warming up account and storage access, respectively.
 $G$ is fully defined in Appendix \ref{app:fees}.
 
+\hypertarget{initcode_cost}{}We define the \textit{initcode cost function}, formally $N$, as the amount of gas that needs to be paid for each word of the initcode prior to executing the creation code of a new contract:
```

Comment:
Since this `EIP-3860` required a "network update", should it indicate that this was introduced with `EIP-3860`?

Reply:
I'm not in favor as it's not a "major" change, but I'm open to discussion.

Reply:
> I'm not in favor as it's not a "major" change, but I'm open to discussion.

Sounds good, don't have strong opinions on this one.